pkg/migrations/op_set_notnull.go

```diff
@@ -162,6 +162,10 @@ func (o *OpSetNotNull) Rollback(ctx context.Context, conn *sql.DB) error {
 }
 
 func (o *OpSetNotNull) Validate(ctx context.Context, s *schema.Schema) error {
+	if err := validateName(o.Column); err != nil {
+		return err
+	}
+
```

Comment:
I don't think it makes sense to check the length of the column name on the operation here as it must already exist in the database. The check can be removed.